src/internal/testutil/debug.go

```diff
@@ -20,7 +21,7 @@ func DebugFiles(t testing.TB, dataRepo string) (map[string]*globlib.Glob, []stri
 	}
 	// Record glob patterns for expected source repo files.
 	for _, file := range []string{"commits.json", "commits-chart**"} {
-		pattern := path.Join("source-repos", dataRepo, file)
+		pattern := path.Join("source-repos", pfs.DefaultProjectName, dataRepo, file)
```

Comment:
I don't know about this.  We basically get no test coverage for dumps that involve projects; probably a bad idea to not be absolutely sure that we can debug projects.

Reply:
I’ve updated the tests to test all four options of auth/no-auth and default-project/non-default-project.